test/services/data_model_engine.js

```diff
@@ -1135,6 +1136,12 @@ describe('Data Model Engine', () => {
         entityBuilder: mockEntityBuilder,
         rolesRepository: mockRolesRepository
       });
+
+      verifyBundleStub = sinon.stub(modelEngine, 'verifyBundle').resolves();
+    });
+
+    afterEach(() => {
+      verifyBundleStub.restore();
```

Comment:
I think restoring stub on object that we create in beforeEach isn't needed, though i'm not sure

Reply:
Just in case